src/sage/interfaces/interface.py

```diff
@@ -1581,18 +1581,18 @@ def _mul_(self, right):
             sage: f = maxima.function('x','sin(x)')
             sage: g = maxima('-cos(x)') # not a function!
             sage: f*g
-            -cos(x)*sin(x)
+            -...cos(x)*sin(x)...
             sage: _(2)
-            -cos(2)*sin(2)
+            -...cos(2)*sin(2)...
```

Comment:
I give up... :fearful:  At some point these tests are just not useful anymore... with all this `...` stuffed in.

Reply:
It's still not a great solution, but if we drop the negative sign from `'-cos(x)`, then the answer is consistent.